_posts/2023-02-03_medical-imaging-with-nix/medical-imaging-with-nix.html

```diff
@@ -86,7 +86,7 @@
 
 
   <!--radix_placeholder_meta_tags-->
-  <title>Reproducible medical imaging software environments in Nix,&lt;/br&gt; or,&lt;/br&gt; Living in the future is hard.</title>
+  <title>Reproducible medical imaging software environments in Nix, or, Living in the future is hard.</title>
```

Comment:
Did the Javascript snippet stop working? 🤔 Looks like the linebreaks were removed

Reply:
the javascript modifies the title after page load, I removed the breaks from title proper so they don't appear in the post index when the blog is rerendered